### ISSUE TITLE
Use x date strings in demos

### ISSUE THREAD

**Issue by TorsteinHonsi** [highcharts/highcharts#22096]
Use x date strings in demos. The `tools/remove-date-utc.js` script replaces most instances of `Date.UTC` from the demos with their string counterparts.

~~This step must be done after #21258 is merged into the master, so that we can benefit from visual compare. But as late as possible before the release, because it will break existing demos when the master is pushed.~~~

Edit: Better to run local visual diffs against v12-staging.
